0000-interledger-eli5/0000-interledger-eli5.md

```diff
@@ -0,0 +1,124 @@
+## Interledger Protocol: Explained Like I'm Five
+
+### Introduction
+
+A _protocol_ is set of rules about how to do something.
+
+The Internet can be seen as a set of rules about how to send messages. For example, when a person follows the rules of the Internet to send an email, it can be said that the person is _using the Internet Protocol to transmit information._
+
+An _asset_ is a word meaning anything valuable. For example, money is an asset.
+
+Interledger (aka _Interledger Protocol_, or _ILP_) is a set of rules about how to send assets. For example, when a person follows the rules of the Interledger to send money, it can be said that the person is _using the Interledger Protocol to transmit an asset._
+
+Interledger, like the Internet, isn't only used by people. It can be used by a computer, a group of computers, or a group of people and/or computers acting together as a company or corporation. The word _party_ is sometimes used to describe a person, thing, or any group of people or things that can _participate in an action._
+
+The Interledger Protocol has four basic building blocks that participate together in the action of transmitting an asset from one party to another.
```

Comment:
It might be good to include in the intro a little more of the benefits of what ILP gives you (payments between people on different networks who use different currencies)